ingestion/src/metadata/sampler/sqlalchemy/bigquery/sampler.py

```diff
@@ -130,62 +130,3 @@ def get_sample_query(self, *, column=None) -> Query:
             )
 
         return super().get_sample_query(column=column)
-
-    def fetch_sample_data(self, columns: Optional[List[Column]] = None) -> TableData:
```

Comment:
the multi project issue was fixed [here](https://github.com/open-metadata/OpenMetadata/pull/20779). This is not needed and was ofuscating the base implementation